apps/spruce/src/analytics/taskQueue/useTaskQueueAnalytics.ts

```diff
@@ -1,11 +1,11 @@
 import { useAnalyticsRoot } from "analytics/useAnalyticsRoot";
 
 type Action =
-  | { name: "Select Distro"; distro: string }
-  | { name: "Click Task Link" }
-  | { name: "Click Version Link" }
-  | { name: "Click Project Link" }
-  | { name: "Click Activated By Link" };
+  | { name: "Changed distro"; distro: string }
+  | { name: "Click task link" }
+  | { name: "Click version link" }
+  | { name: "Click project link" }
+  | { name: "Click activated by link" };
```

Comment:
I think we are going with past tense? `Click` ➡️ `Clicked`

Looks like you could even update `name` within `ActionType` to assert that value starts with one of the tokens we're using: https://stackoverflow.com/questions/68846567/typescript-string-type-that-must-start-with-specific-characters

Reply:
WOW great find!